src/server/models/listItem/listItem.ts

```diff
@@ -350,15 +343,14 @@ export async function update(
   // @ts-ignore
   const data: DeserialisedWebhookData | null | undefined = legacyDataParameter ?? jsonData?.updatedJsonData;
 
-  if (!data) {
-    logger.error(
+  if (!legacyDataParameter ?? jsonData?.updatedJsonData) {
+    logger.warn(
       "listItem.update cannot resolve any data to update the list item with jsonData.updatedJsonData and data parameter were empty"
     );
-    throw Error(`${userId} attempted to update ${id} but no data was found`);
```

Comment:
previously this was throwing an error, but in some cases (like a provider does not change their info for annual review), the data object will be empty.